<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ajouter_Client</class>
 <widget class="QDialog" name="Ajouter_Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources/ressources.qrc">
     <normaloff>:/Back.png</normaloff>:/Back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>150</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_iddemande">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources/ressources.qrc">:/gestion clients.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenom">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cin">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>210</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nat">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_ddn">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>290</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>380</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nbvol">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>1278</x>
     <y>210</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_ddi">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>460</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_affiche">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>660</y>
     <width>1581</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_recherche">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>420</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_iddemande">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>260</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimer">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>510</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>740</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ordre Alphabetique</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nombre De Vols</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Identifiant</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Date D'Inscription</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_modifier">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>320</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_valider">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <zorder>label_iddemande</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>lineEdit_nom</zorder>
  <zorder>lineEdit_prenom</zorder>
  <zorder>lineEdit_cin</zorder>
  <zorder>lineEdit_nat</zorder>
  <zorder>dateEdit_ddn</zorder>
  <zorder>lineEdit_contact</zorder>
  <zorder>lineEdit_nbvol</zorder>
  <zorder>lineEdit_id</zorder>
  <zorder>dateEdit_ddi</zorder>
  <zorder>tableView_affiche</zorder>
  <zorder>pushButton_recherche</zorder>
  <zorder>lineEdit_iddemande</zorder>
  <zorder>pushButton_supprimer</zorder>
  <zorder>comboBox</zorder>
  <zorder>pushButton_modifier</zorder>
  <zorder>pushButton_valider</zorder>
 </widget>
 <resources>
  <include location="resources/ressources.qrc"/>
  <include location="resources/ressources.qrc"/>
  <include location="resources/ressources.qrc"/>
  <include location="resources/ressources.qrc"/>
  <include location="resources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
